pipeline.py

```diff
@@ -96,21 +94,35 @@ def process_file(path, output):
     img = nibabel.load(path)
     if affine_matrix_permutes_axes(img.affine):
         return
-    processed_file = get_tempfile_name()
-    os.system('/usr/lib/ants/N4BiasFieldCorrection -i %s -o %s -s 6' %
-              (path, processed_file))
-    img = nibabel.load(processed_file)
+
     array = img.get_fdata()
     array = np.nan_to_num(array)
     std = np.std(array.reshape(-1))
+
     array = array / std
     mean = np.mean(array.reshape(-1))
     # HACK Alert - This is a way to check if the backgound is a white noise.
     if mean > 1.0:
         print('mean too high: %s' % mean)
         return
-    array -= mean
-    for k in range(array.shape[2]):
+
+    processed_file = get_tempfile_name()
+    os.system('/usr/lib/ants/N4BiasFieldCorrection -i %s -o %s -s 6' %
```

Comment:
can we add the skull-stripping with a similar line?

Reply:
I think so but skull stripping might take ages. I can add a todo